atomics/T1555.003/T1555.003.yaml

```diff
@@ -412,3 +412,45 @@ atomic_tests:
       Stop-Process -Name "WebBrowserPassView"
     name: powershell
     elevation_required: true 
+- name: BrowserStealer (Chrome / Firefox / Microsoft Edge)
+  description: |
+    [Github Repo](https://github.com/SaulBerrenson/BrowserStealer) Simple password/cookies stealer for chrome, edge, and gecko based browsers (30 listed working). This attack simulates stealing the data from the browser files and printing them to the command line.
+    If using to test with Firefox, if the browser is x64 you need to use the x64 build
+  supported_platforms:
+  - windows
+  dependency_executor_name: powershell
+  dependencies:
+  - description: |
+      Google Chrome must be on the device.
+    prereq_command: |
+      'if ((Test-Path "C:\Program Files\Google\Chrome\Application\chrome.exe") -Or (Test-Path "C:\Program Files (x86)\Google\Chrome\Application\chrome.exe")) {exit 0} else {exit 1}'
+    get_prereq_command: |
+      $installer = "$env:temp\ChromeStandaloneSetup64.msi"
+      Invoke-WebRequest -OutFile $env:temp\ChromeStandaloneSetup64.msi https://dl.google.com/chrome/install/googlechromestandaloneenterprise64.msi
+      msiexec /i $installer /qn
+      Start-Process -FilePath "chrome.exe"
+      Stop-Process -Name "chrome"
+  - description: |
+      BrowserCollector must exist in the temp directory
+    prereq_command: |
+      if (Test-Path "PathToAtomicsFolder\T1555.003\bin\BrowserCollector.exe") {exit 0} else {exit 1}
+    get_prereq_command: |-
+      Invoke-WebRequest "https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T155.003/bin/BrowserCollector.exe?raw=true" -Outfile: "PathToAtomicsFolder\T1555.003\bin\BrowserCollector.exe"
```

Comment:
Instead of adding the BrowserCollector.exe as part of this PR could you instead pull it from the releases page for the original authors repo? https://github.com/SaulBerrenson/BrowserStealer/releases

This help to show where the tools comes from, what the source code is, and avoids any licensing issues. Thank you.